src/Coduo/PHPHumanizer/String/TextTruncate.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Coduo\PHPHumanizer\String;
+
+class TextTruncate implements Truncate
+{
+    /**
+     * @var string
+     */
+    private $text;
+
+    /**
+     * @var int
+     */
+    private $charactersCount;
+
+    /**
+     * @var string
+     */
+    private $append;
+
+    /**
+     * @var Breakpoint
+     */
+    private $breakpoint;
+
+    /**
+     * @param string $text
+     * @param int    $charactersCount
+     * @param string $append
+     */
+    public function __construct($text, $charactersCount, $append = '')
+    {
+        $this->text            = $text;
+        $this->charactersCount = $charactersCount;
+        $this->append          = $append;
+    }
+
+    /**
+     * @param Breakpoint $breakpoint
+     * @return TextTruncate
+     */
+    public function setBreakpoint($breakpoint) {
```

Comment:
This should be passed via constructor cuz without that object TextTruncate have no sense.